src/Compilers/Core/VBCSCompiler/App.config

```diff
@@ -5,6 +5,18 @@
   <runtime>
     <gcServer enabled="true"/>
     <gcConcurrent enabled="false"/>
+    <dependentAssembly>
+        <assemblyIdentity name="Microsoft.CodeAnalysis" publicKeyToken="31bf3856ad364e35" culture="neutral" />
+        <bindingRedirect oldVersion="1.0.0.0-1.0.65535.65535" newVersion="2.0.0.0" />
```

Comment:
@agocke Presumably we would want to redirect a binary built against any 1.* to be redirected. That is, shouldn't we be doing `oldVersion="1.0.0.0-1.65535.65535.65535"`?

Also, it looks like your formatting is a little off. Most of the file uses two spaces for indentation, not four.